po/LINGUAS

```diff
@@ -1,8 +0,0 @@
-en
-fr
-pl
```

Comment:
I think this is easier for translators to add new languages in here than it is to edit main.rs. Maybe include_i18! should read this file instead of getting languages from a static parameter (actually it could accept both), what do you think @BaptisteGelez ?

Reply:
Ideally, I think include_i18n! should read all the .mo files in `translations` if no language list is provided. This way new languages would be loaded automatically, without needing to add them anywhere.

Reply:
It seems like it's not possible to `ls` with normal macros, however it exist a crate doing more or less what we want [here](https://github.com/pyros2097/rust-embed) (we could also use it to embed wasm files later on)

Reply:
I think using any of those isn't in the range of this pr anyway, a macro is a thing, a dependency is another